### ISSUE TITLE
Incorporate into HexGui?

### ISSUE THREAD

**Issue by Twixtfanatic** [cgao3/benzene-vanilla-cmake#2]
Thank you for fixing up the mess the Benzene project was in. I was able to build under Linux Mint 19 with no difficulties.

Do you have any plan to incorporate MoHex into a gui such as Broderick Arneson's HexGui? I believe that was once part of Benzene. Here is a link to a screenshot.

https://cf.geekdo-images.com/imagepagezoom/img/EtB141K5lmoAEejqCUOaq9GPK04=/fit-in/1200x900/filters:no_upscale()/pic4247242.png

Thanks for your time.

**Comment by cgao3** [comment]
yes, HexGUI supports MoHex, but seems you don't know how to load MoHex from HexGUI. 

I can write a section in the README on how to use benzene with HexGUI.

**Comment by Twixtfanatic** [comment]
That would help, thank you. Maybe I should put them in the same directory?

BTW do you like the board display with triangles?

**Comment by cgao3** [comment]
I have updated the document, please refer to "HexGUI" section for how to connect your program to HexGUI. 

In your screenshot, displaying with triangle looks nice.

**Comment by cgao3** [comment]
echo "PATH=PATH:`pwd`/hexgui" >> ~/.bashrc

I forget to append /hexgui...

Anyway, you should be able to run the executable hexgui/bin/hexgui directly, right? 

HexGUI supports go-style board already..
